jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridPresenter.java

```diff
@@ -162,70 +177,184 @@ public void getData( final Range visibleRange ) {
-                                            dataSetQueryHelper.getColumnStringValue( dataSet, DataSetTasksListGridViewImpl.COLUMN_CREATEDBY, i ),
-                                            dataSetQueryHelper.getColumnDateValue( dataSet, DataSetTasksListGridViewImpl.COLUMN_CREATEDON, i ),
-                                            dataSetQueryHelper.getColumnDateValue( dataSet, DataSetTasksListGridViewImpl.COLUMN_ACTIVATIONTIME, i ),
-                                            dataSetQueryHelper.getColumnDateValue( dataSet, DataSetTasksListGridViewImpl.COLUMN_DUEDATE, i ),
-                                            dataSetQueryHelper.getColumnStringValue( dataSet, DataSetTasksListGridViewImpl.COLUMN_PROCESSID, i ),
-                                            dataSetQueryHelper.getColumnLongValue( dataSet, DataSetTasksListGridViewImpl.COLUMN_PROCESSSESSIONID, i ),
-                                            dataSetQueryHelper.getColumnLongValue( dataSet, DataSetTasksListGridViewImpl.COLUMN_PROCESSINSTANCEID, i ),
-                                            dataSetQueryHelper.getColumnStringValue( dataSet, DataSetTasksListGridViewImpl.COLUMN_DEPLOYMENTID, i ),
-                                            dataSetQueryHelper.getColumnLongValue( dataSet, DataSetTasksListGridViewImpl.COLUMN_PARENTID, i ) ) );
-
-                                }
-                                PageResponse<TaskSummary> taskSummaryPageResponse = new PageResponse<TaskSummary>();
-                                taskSummaryPageResponse.setPageRowList( myTasksFromDataSet );
-                                taskSummaryPageResponse.setStartRowIndex( visibleRange.getStart() );
-                                taskSummaryPageResponse.setTotalRowSize( dataSet.getRowCountNonTrimmed() );
-                                taskSummaryPageResponse.setTotalRowSizeExact( true );
-                                if ( visibleRange.getStart() + dataSet.getRowCount() == dataSet.getRowCountNonTrimmed() ) {
-                                    taskSummaryPageResponse.setLastPage( true );
-                                } else {
-                                    taskSummaryPageResponse.setLastPage( false );
-                                }
-                                DataSetTasksListGridPresenter.this.updateDataOnCallback( taskSummaryPageResponse );
-                            }
-                            view.hideBusyIndicator();
-                        }
+                    dataSetQueryHelper.setDataSetHandler(currentTableSettings);
+                    dataSetQueryHelper.lookupDataSet(visibleRange.getStart(), createDataSetTaskCallback(visibleRange.getStart(), currentTableSettings));
+
+                }
+            }
+        } catch ( Exception e ) {
+            GWT.log( "Error looking up dataset with UUID [ jbpmHumanTasks ]" );
```

Comment:
Please display the error message using errorPopup.showMessage.
Overall I think GWT.log could be removed from all catch statements, keeping only errorPopup as primary way to indicate an error to the user.